framework/src/modules/chain/blocks/block_slots.js

```diff
@@ -115,6 +115,16 @@ class BlockSlots {
 		return nextSlot + this.blocksPerRound;
 	}
 
+	/**
+	 * Check if timestamp is contained within a slot time window
+	 * @param slot
+	 * @param time
+	 * @returns {boolean}
+	 */
+	timeFallsInSlot(slot, time) {
```

Comment:
Maybe better naming?
`IsWithinTimeslot` or something that indicates it returns boolean?

Reply:
Definitely better, thanks :)